src/components/content/heroes/aurora/weather-easter-egg/rain.tsx

```diff
@@ -0,0 +1,58 @@
+import { Flare } from '../flare';
+import PrecipitationEffect, { PrecipitationType } from './precipitation-effect';
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { DefaultFlares, DefaultFlaresDark } from '../default-flares';
+
+export const AuroraRainyFlareColor = styled.div`
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(180deg, #231f67 0%, #4d79ff45 90%);
+  position: absolute;
+`;
+
+interface RainProps {
+  amount: number;
+}
+
+export const Rain: FC<RainProps> = ({ amount }) => {
```

Comment:
My favourite effect. Only the drops splashing at the ground are missing to be super awesome and not only awesome 😁
![CleanShot 2023-07-20 at 09 27 09](https://github.com/satellytes/satellytes.com/assets/1701755/ba6a49c6-8582-493e-bd6a-8f58be7c9cdd)

Reply:
added